File: ProjetPI/src/ProjectBundle/Entity/Logement.php
<?php

namespace ProjectBundle\Entity;

use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;

/**
 * Logement
 *
 * @ORM\Table(name="logement")
 * @ORM\Entity(repositoryClass="ProjectBundle\Repository\LogementRepository")
 */
class Logement
{
    /**
     * @var int
     *
     * @ORM\Column(name="id_log", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id_log;

    /**
     * @return int
     */
    public function getIdLog()
    {
        return $this->id_log;
    }

    /**
     * @return \AppBundle\Entity\User
     */
    public function getIdUser()
    {
        return $this->id_user;
    }

    /**
     * @param \AppBundle\Entity\User $id_user
     */
    public function setIdUser($id_user)
    {
        $this->id_user = $id_user;
    }

    /**
     * @param int $id_log
     */

    /**
     * @ORM\ManyToOne(targetEntity=ProjectBundle\Entity\Refugies::class)
     * @ORM\JoinColumn(name="id_ref",referencedColumnName="id_ref")
     */
    private $id_ref;

    /**
     * @var \AppBundle\Entity\User
     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\User")
     *
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_user", referencedColumnName="id", nullable=true , onDelete="CASCADE")
     * })
     */
    private $id_user;

    public function setIdLog($id_log)
    {
        $this->id_log = $id_log;
    }

    /**
     * @return mixed
     */
    public function getIdRef()
    {
        return $this->id_ref;
    }

    /**
     * @param mixed $id_ref
     */
    public function setIdRef($id_ref)
    {
        $this->id_ref = $id_ref;
    }

    /**
     * @return mixed
     */
    public function getNomLog()
    {
        return $this->nomLog;
    }

    /**
     * @param mixed $nomLog
     */
    public function setNomLog($nomLog)
    {
        $this->nomLog = $nomLog;
    }

    /**
     * @return string
     */
    public function getAdresse()
    {
        return $this->adresse;
    }

    /**
     * @param string $adresse
     */
    public function setAdresse($adresse)
    {
        $this->adresse = $adresse;
    }

    /**
     * @return string
     */
    public function getEtatLog()
    {
        return $this->etatLog;
    }

    /**
     * @param string $etatLog
     */
    public function setEtatLog($etatLog)
    {
        $this->etatLog = $etatLog;
    }

    /**
     * @return int
     */
    public function getNbChambre()
    {
        return $this->nbChambre;0
    }

    /**
     * @param int $nbChambre
     */
    public function setNbChambre($nbChambre)
    {
        $this->nbChambre = $nbChambre;
    }


    /**
     * @var string
     *
     * @Assert\Type("alpha",
     * message="ce champs doit etre de type string ")
     * @ORM\Column(name="nom_log", type="string", length=255)
     */
    private $nomLog;

    /**
     * @var string
     *
     * @Assert\Type("alpha",
     * message="ce champs doit etre de type string ")
     * @ORM\Column(name="adresse", type="string", length=255)
     */
    private $adresse;

    /**
     * @var string
     *@Assert\Type("alpha",
     * message="ce champs doit etre de type string ")
     * @ORM\Column(name="etat_log", type="string", length=255)
     */
    private $etatLog;

    /**
     * @var int
     *
     * @Assert\GreaterThan(value=0,
     * message="Le nombre doit etre supérieur à 0 "
     * )
     *
     *
     *
     * @ORM\Column(name="nb__chambre", type="integer")
     */
    private $nbChambre;







}

File: pimobile/src/entities/Logement.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entities;

import java.util.Objects;

/**
 *
 * @author dEll
 */
public class Logement {
    
    private int nb__chambre;
    private int id_log;
    private int id_user;
    private String adresse;
    private String etat_log;
    private int id_ref;
    private String nom_log;

    public Logement(int nb__chambre, int id_log, int id_user, String adresse, String etat_log, int id_ref, String nom_log) {
        this.nb__chambre = nb__chambre;
        this.id_log = id_log;
        this.id_user = id_user;
        this.adresse = adresse;
        this.etat_log = etat_log;
        this.id_ref = id_ref;
        this.nom_log = nom_log;
    }

    public Logement() {
    }

    public Logement(int nb__chambre, String adresse, String etat_log, String nom_log) {
        this.nb__chambre = nb__chambre;
        this.adresse = adresse;
        this.etat_log = etat_log;
        this.nom_log = nom_log;
    }

    public Logement(int nb__chambre, String adresse, String nom_log) {
        this.nb__chambre = nb__chambre;
        this.adresse = adresse;
        this.nom_log = nom_log;
    }
    
    
    public int getNb__chambre() {
        return nb__chambre;
    }

    public void setNb__chambre(int nb__chambre) {
        this.nb__chambre = nb__chambre;
    }

    public int getId_log() {
        return id_log;
    }

    public void setId_log(int id_log) {
        this.id_log = id_log;
    }

    public int getId_user() {
        return id_user;
    }

    public void setId_user(int id_user) {
        this.id_user = id_user;
    }

    public String getAdresse() {
        return adresse;
    }

    public void setAdresse(String adresse) {
        this.adresse = adresse;
    }

    public String getEtat_log() {
        return etat_log;
    }

    public void setEtat_log(String etat_log) {
        this.etat_log = etat_log;
    }

    public int getId_ref() {
        return id_ref;
    }

    public void setId_ref(int id_ref) {
        this.id_ref = id_ref;
    }

    public String getNom_log() {
        return nom_log;
    }

    public void setNom_log(String nom_log) {
        this.nom_log = nom_log;
    }

    @Override
    public int hashCode() {
        int hash = 7;
        hash = 97 * hash + this.nb__chambre;
        hash = 97 * hash + this.id_log;
        hash = 97 * hash + this.id_user;
        hash = 97 * hash + Objects.hashCode(this.adresse);
        hash = 97 * hash + Objects.hashCode(this.etat_log);
        hash = 97 * hash + this.id_ref;
        hash = 97 * hash + Objects.hashCode(this.nom_log);
        return hash;
    }

    @Override
    public boolean equals(Object obj) {
        if (this == obj) {
            return true;
        }
        if (obj == null) {
            return false;
        }
        if (getClass() != obj.getClass()) {
            return false;
        }
        final Logement other = (Logement) obj;
        if (this.nb__chambre != other.nb__chambre) {
            return false;
        }
        if (this.id_log != other.id_log) {
            return false;
        }
        if (this.id_user != other.id_user) {
            return false;
        }
        if (this.id_ref != other.id_ref) {
            return false;
        }
        if (!Objects.equals(this.adresse, other.adresse)) {
            return false;
        }
        if (!Objects.equals(this.etat_log, other.etat_log)) {
            return false;
        }
        if (!Objects.equals(this.nom_log, other.nom_log)) {
            return false;
        }
        return true;
    }

    @Override
    public String toString() {
        return "Logement{" + "nb__chambre=" + nb__chambre + ", id_log=" + id_log + ", id_user=" + id_user + ", adresse=" + adresse + ", etat_log=" + etat_log + ", id_ref=" + id_ref + ", nom_log=" + nom_log + '}';
    }

   
    
    
    
}
